less/index.less

```diff
@@ -1 +1,53 @@
 // Start here!
+
+
+
+.container{
+    background-color: rgb(31, 27, 27);
+    //background-image: url("https://goo.gl/pnTSRP");
+    //background-position: center;
+    
+    .jumbotron{
+        background-color: rgb(31, 27, 27,0.2);
```

Comment:
it's easy when all of your colours are referred to right next to each other, like you can clearly see that the rgb is correct, but in general it's recommended that you use a variable for colour values like this, just so it's easy to remain consistent all the way throughout the project.